docs/source/Installation.rst

```diff
@@ -33,108 +48,95 @@ Package-based Process
 -  Create a new environment with the caiman package from conda-forge:
--  If you are installing on Windows, use the conda enabled shell (under "Anaconda" in your programs menu) rather than powershell or a generic prompt
+-  If you are installing on Windows, use the conda enabled shell (under "Anaconda" in your programs menu) rather than powershell or a generic prompt:
 
 .. code:: bash
 
     mamba create -n caiman -c conda-forge caiman
     conda activate caiman
 
--  Skip ahead to the section on setting up a data directory with caimanmanager
+- The previous two steps installed caiman and activated the virtual environment. 
+-  Skip ahead to the section on setting up a data directory with `caimanmanager`
 
-Known issues
-~~~~~~~~~~~~
+**Known issues**
 
-If you are on Windows, have used CaImAn before using our github repo and now want to use the conda-forge package,
-you might encounter some errors with Python reading the files from the wrong directory. In this case rename
-(or remove) the caiman directory that contains the source of the repo and the caiman_data folder and then proceed
-with setting up the caiman_data folder as explained below.
+If you are on Windows, have used Caiman before using our github repo and now want to use the conda-forge package,
+you might encounter some errors with Python reading the files from the wrong directory. In this case, rename
+(or remove) the caiman directory that contains the source of the repo and the ``caiman_data`` folder and then proceed
+with setting up the ``caiman_data`` folder as explained below in Section 2.
 
 .. raw:: html
 
    </details>
 
 
-Development mode Installation Process
--------------------------------------
+Section 1B. Development-mode install
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This will allow you to modify the source files of CaImAn and will make it easier
-to contribute to the CaImAn project, fix bugs etc.
+Dev mode install allows you to modify the source files of Caiman and makes it easier
+to contribute to the project, fix bugs etc.. The general motivation for setting up
+an editable development environment is described in detail in our `contributors page <../../CONTRIBUTING.md>`_.
 
 
-Installation on Windows
-~~~~~~~~~~~~~~~~~~~~~~~
+Dev-mode install on Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. raw:: html
 
    <details>
-   <summary>Installation on Windows</summary>
+   <summary>Dev Mode Installation on Windows</summary>
 
 The Windows installation process differs more widely from installation
-on Linux or MacOSX and has different issues you may run into. We strongly recommend
-using the package-based install rather than building CaImAn yourself. In either case,
-everything you do should be from a Conda enabled shell (which you can find in your programs menu)
-rather than from Powershell or any other shell.
+on Linux or MacOSX and has different issues you may run into. Everything 
+you do should be from the anaconda prompt rather than from Powershell 
```

Comment:
The user may or may not have two icons, one for anaconda shell and one for anaconda powershell. We should make sure this isn't confusing if they have both.

Reply:
Good point: reverting to the earlier language to make this clear.